terraform/vars.tf

```diff
@@ -1,5 +1,10 @@
 terraform {
   required_version = "> 0.11"
+
+  backend "gcs" {
+    bucket = "dev-atm-data-lake-drain-tf-state"
```

Comment:
Please make it configurable during the initialisation as described in https://www.terraform.io/docs/backends/config.html

Reply:
Done